recipes/cuda-compat/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set name = "cuda-compat" %}
+{% set version = "525.60.13" %}
+{% set cuda_version = "12.0.0" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set extension = "tar.xz" %}  # [not win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ cuda_version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/nvidia_driver/{{ platform }}/nvidia_driver-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: a42691822f6e87060da5682a8e0662c86d1b20fbf31d12cb2969b819995e4138  # [linux64]
+  sha256: 4ad08f8431195580349f4984fae2d93697d2f5572837c1e7ae59c37feab291c8  # [ppc64le]
+  sha256: 1708ae3ba857559cbfc4e6c06267ed6126eff2c68fa5321a95906f50e984148c  # [aarch64]
+
+build:
+  number: 0
+  skip: true  # [osx or win]
+  script_env:
+    - DRV_VERSION={{ version }}
+
+outputs:
+  - name: cuda-compat
+    requirements:
+      run:
+        - {{ pin_subpackage("cuda-compat-impl", exact=True) }}
+    test:
+      commands:
+        - exit 0
+    about:
+      home: https://developer.nvidia.com/cuda-toolkit
+      license_file: LICENSE
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_url: https://docs.nvidia.com/cuda/eula/index.html
+      summary: CUDA Compatibility Platform
+      description: |
+          Package containing all the necessary CUDA driver files related to forward
+          compatibility. This compatibility enables newer CUDA runtimes to work with
+          older display drivers. Used for Tesla cards only.
+      doc_url: https://docs.nvidia.com/cuda/index.html
+
+  - name: cuda-compat-impl
+    version: {{ version }}
```

Comment:
Were we planning to have a "dual version" here, for both CTK and driver versions? I'm not seeing both `version` (driver) and `cuda_version` (CTK version) being used in the impl/non-impl packages. Maybe I'm missing something.

Reply:
Both versions are being used, `cuda-compat` uses `cuda_version` [here](https://github.com/conda-forge/staged-recipes/pull/21952/files#diff-688d8da022bb4c13e857bc25239b355421f8d5ff0fd0845ba2299ccab64abf45R11)

And here are the artifacts:
```
    /home/conda/staged-recipes/build_artifacts/linux-64/cuda-compat-impl-525.60.13-h59595ed_0.conda \
    /home/conda/staged-recipes/build_artifacts/linux-64/cuda-compat-12.0.0-ha770c72_0.conda
```

Reply:
Should we rename `version` to `driver_version` for clarity?

Reply:
Sure, renamed it in the latest.